definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState.json

```diff
@@ -1670,7 +1670,7 @@
     "CaseTypeID": "DIVORCE",
     "CaseStateID": "AosSubmittedAwaitingAnswer",
     "UserRole": "caseworker-divorce-solicitor",
-    "CRUD": "R"
+    "CRUD": "RU"
   },
```

Comment:
The following events above that didn't have the required permission now have 'U' roles added to enable them become visible to the solicitor